src/config/index.js

```diff
@@ -3,4 +3,6 @@ const registerConfig = require('windbreaker-service-util/config')
 const Config = require('./Config')
 const config = module.exports = new Config()
 
-registerConfig(config)
+registerConfig(config, [
+  { amqUrl: `amqp://localhost:${process.env.AMQ_PORT || 5672}` }
+])
```

Comment:
Not sure if this is the typical pattern for config/model or if environment variable overrides generally happen elsewhere. Let me know if anyone has a preferred alternative. I noticed @charlieduong94 started `server-config.yml` but it doesn't seem to be used yet. Maybe that was intended for overrides in some way?